### PR TITLE
Added in link containers for photocards

### PR DESCRIPTION
**Pull request by Espe-on**

### CODE REVIEW

**Comment by Espe-on** [comment]
Have added containers for each Photocard in Discover Rovers page.

File: src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
-            <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-spirit-opportunity/en/spirit-opportunity.en.jpg"} altText={"Spirit and Opportunity Rovers"} imageTitle={"Spirit and Opportunity Rovers"} description={"Spirit and Opportunity are twin rovers that were made to learn more about the planet Mars."}/>
-            <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-curiosity/en/curiosity-here.en.jpg"} altText={"Curiosity Rover"} imageTitle={"Curiosity Rover"} description={"Curiosity is a rover that was sent to Mars to determine if the Red Planet ever had the proper conditions for microbial life to survive."}/>
+            <Link to={`${window.location.pathname}/sojourner`}>
+                <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-sojourner/en/sojourner-first.en.jpg"}
+                           altText={"Sojourner Rover"}
+                           imageTitle={"Sojourner Rover"}
+                           description={"Sojourner is a rover that landed on Mars in 1997 in a location called Ares Vallis where it explored and took many photos."}/>
```
Comment (MichaelWalker) [inline src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx:15]: Is this Photocard ever used without being a link?
If not, would it be nicer to push the link down into the Photocard component?

File: src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+export function DiscoverRoversPage() {
     return (
         <div className="discoverRovers">
             <h1>Discover Rovers</h1>
             <h3>Meet the Rovers on The Red Planet</h3>
-            <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-sojourner/en/sojourner-first.en.jpg"} altText={"Sojourner Rover"} imageTitle={"Sojourner Rover"} description={"Sojourner is a rover that landed on Mars in 1997 in a location called Ares Vallis where it explored and took many photos."}/>
-            <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-spirit-opportunity/en/spirit-opportunity.en.jpg"} altText={"Spirit and Opportunity Rovers"} imageTitle={"Spirit and Opportunity Rovers"} description={"Spirit and Opportunity are twin rovers that were made to learn more about the planet Mars."}/>
-            <Photocard imageUrl={"https://spaceplace.nasa.gov/mars-curiosity/en/curiosity-here.en.jpg"} altText={"Curiosity Rover"} imageTitle={"Curiosity Rover"} description={"Curiosity is a rover that was sent to Mars to determine if the Red Planet ever had the proper conditions for microbial life to survive."}/>
+            <Link to={`${window.location.pathname}/sojourner`}>
```
Comment (MichaelWalker) [inline src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx:11]: Do we not just want these links to be
`/sojourner`
`.curiosity`
etc

What is the `window.location.pathname` for?

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
A couple of questions, but otherwise looks good.

Very happy to be convinced if I am talking nonsense! :)

File: src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx (hunk 2 above)
Comment (Espe-on) [reply src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx:11]: It means that they are nested under the current page

Comment (Espe-on) [reply src/Pages/DiscoverRoversPage/DiscoverRoversPage.tsx:11]: so it makes the link under the page the user is on

**Review by MichaelWalker — APPROVED** [review]

**Pull request merged**